packages/dashboard/src/redux/reducers/trade.js

```diff
@@ -70,6 +74,24 @@ const tradeState = (prevState = initialState, { type, payload }) => {
       }
     }
 
+    case GET_WATCHLIST: {
+      return {
+        ...prevState,
+        isWatchlistLoading: true,
+      }
+    }
+
+    case GET_WATCHLIST_RESPONSE: {
+      return {
+        ...prevState,
+        isWatchlistLoading: false,
+        watchlist: {
+          ...prevState.watchlist,
```

Comment:
I don't think we need this. I only did this on  `historicQuotes` so we keep all previous historic quotes and only overwrite the new one we get back.